src/functions/leaderboard/leaderboard.go

```diff
@@ -0,0 +1,33 @@
+package leaderboard
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"strings"
+)
+
+const BASE_URL = "https://archybot.web.app/leaderboard/"
```

Comment:
All variables are camelCase or PascalCase, this should be `baseUrl`

Reply:
Alright! I will fix it